.gitignore

```diff
@@ -3,3 +3,10 @@ dist/
 node_modules
 .husky
 .turbo
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
Recommended via https://yarnpkg.com/getting-started/qa#:~:text=yarn%2Finstall%2Dstate.,your%20workspaces%20all%20over%20again., which also suggests not committing `install-state.gz`, but I might be causing CI issues.